clinica/iotools/converters/adni_to_bids/adni_modalities/adni_fmri.py

```diff
@@ -130,21 +129,33 @@ def _compute_fmri_path(
     fmri_dfs_list = []
 
     # Loading needed .csv files
-    adni_merge = load_clinical_csv(csv_dir, "ADNIMERGE")
-
-    mayo_mri_qc = load_clinical_csv(csv_dir, "MAYOADIRL_MRI_IMAGEQC_12_08_15")
-
+    # adni_merge_path = path.join(csv_dir, "ADNIMERGE.csv")
+    # adni_merge = pd.read_csv(adni_merge_path, delimiter='","')
+    # adni_merge.columns = adni_merge.columns.str.strip('"')
+    adni_merge = pd.read_csv(path.join(csv_dir, "ADNIMERGE2.csv"), sep=",", engine='python')
+
+    mayo_mri_qc = pd.read_csv(
+        path.join(csv_dir, "MAYOADIRL_MRI_IMAGEQC_12_08_15.csv"),
+        sep=",",
+        low_memory=False,
+    )
     mayo_mri_qc = mayo_mri_qc[mayo_mri_qc.series_type == "fMRI"]
     mayo_mri_qc.columns = [x.upper() for x in mayo_mri_qc.columns]
 
-    mayo_mri_qc3 = load_clinical_csv(csv_dir, "MAYOADIRL_MRI_QUALITY_ADNI3")
+    mayo_mri_qc3 = pd.read_csv(
+        path.join(csv_dir, "MAYOADIRL_MRI_QUALITY_ADNI3.csv"), sep=",", low_memory=False
+    )
     mayo_mri_qc3 = mayo_mri_qc3[mayo_mri_qc3.SERIES_TYPE == "EPB"]
 
     # Concatenating visits in both QC files
     mayo_mri_qc = pd.concat(
         [mayo_mri_qc, mayo_mri_qc3], axis=0, ignore_index=True, sort=False
     )
-    mri_list = load_clinical_csv(csv_dir, "MRILIST")
+
+    mri_list = pd.read_csv(path.join(csv_dir, "MRILIST.csv"), sep=",", low_memory=False)
+
+
+    from clinica.utils.stream import cprint
```

Comment:
Was this import missing ?

Reply:
The import was not missing, I was previously using it for debugging, it can be removed